src/bower_components/apiclient/connectionManager.js

```diff
@@ -0,0 +1,1292 @@
+define(["events", "apiclient", "appStorage", "credentialprovider", "apphost"], function (events, apiClientFactory, appStorage, credentialProvider, apphost) {
+    "use strict";
+
+    function getServerAddress(server, mode) {
+        switch (mode) {
```

Comment:
If you made any other changes besides deminification here then you might want to mirror them in the ES6 repo so we can migrate with less issues.

Reply:
I modified this file to reflect some module resolution problems with this module. I can look into it and make a PR with the changes i made here into the connectionManager.js in the apiclient repo.